pkgs/top-level/ocaml-packages.nix

```diff
@@ -751,6 +751,8 @@ let
 
     pgocaml_ppx = callPackage ../development/ocaml-modules/pgocaml/ppx.nix {};
 
+    ocaml_postgresql = callPackage ../development/ocaml-modules/postgresql {};
```

Comment:
Please use `postgresql` as attribute name (i.e., the same as the name of the directory and the same as the name used by `dune` and `ocamlfind`). There are other ways to avoid infinite recursion than using a surprising name; see `gnuplot` for instance.